bitnami/mongodb/Chart.yaml

```
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/bitnami-docker-mongodb
   - https://mongodb.org
-version: 10.26.4
+version: 10.26.5
```

Is a new feature so I think is better:
```suggestion
version: 10.27.0
```